helm/values.yaml

```diff
@@ -33,7 +33,7 @@ notary:
   # if notary uses an authentication server, give a user and password
   # with pull rights
   auth:
-    enabled: true
+    enabled: false
```

Comment:
Why do we want to change the default here?

Reply:
Mainly for the reason that the default creds do not work, so in case you need to authorize you have to touch this part anyways. In case, you use a publicly accessible repo, you do not even have to touch on this :shrug: 
Also, it makes the demo shorter ;-)

Reply:
I think I would add a comment above explaining that :) Just to point out that this is not an insecure default

Reply:
great idea!